stubs/Werkzeug/werkzeug/wrappers.pyi

```diff
@@ -148,7 +148,7 @@ class BaseResponse:
         self,
         key: str,
         value: Union[str, bytes] = ...,
-        max_age: Optional[int] = ...,
+        max_age: Union[int, timedelta, None] = ...,
```

Comment:
```suggestion
        max_age: Union[float, timedelta, None] = ...,
```
This calls `werkzeug.http.dump_cookie()`, which according to our stubs accepts a float